projekt_do_pracy_na_labach/fiszki_app/Dockerfile

```diff
@@ -18,7 +18,7 @@ RUN flutter config --enable-web
 RUN mkdir /app/
 COPY . /app/
 WORKDIR /app/
-RUN flutter build web --dart-define BACKEND_URL=http://backend-service
+RUN flutter build web --dart-define BACKEND_URL=backend:8000
```

Comment:
Jak wykomentujesz `backend_URL` to będzie działać. To jest taki trochę grzech z układania kolejnych labów z kubernetesem i tutaj nie powinno go byc jednak xd